File: SpringBoot/shop-founders-modules/shop-founders-core/src/main/java/hf/shopfounders/model/RemoveLikeResponse.java
package hf.shopfounders.model;

import com.fasterxml.jackson.annotation.JsonProperty;

public class RemoveLikeResponse {

    @JsonProperty("userId")
    private String userId;
    @JsonProperty("shopId")
    private String shopId;
    @JsonProperty("likeStatus")
    private RemoveLikeStatus likeStatus;

    public RemoveLikeResponse() {
    }

    public RemoveLikeResponse(String userId, String shopId, RemoveLikeStatus likeStatus) {
        this.userId = userId;
        this.shopId = shopId;
        this.likeStatus = likeStatus;
    }

    public String getUserId() {
        return userId;
    }

    public void setUserId(String userId) {
        this.userId = userId;
    }

    public String getShopId() {
        return shopId;
    }

    public void setShopId(String shopId) {
        this.shopId = shopId;
    }

    public RemoveLikeStatus getLikeStatus() {
        return likeStatus;
    }

    public void setLikeStatus(RemoveLikeStatus likeStatus) {
        this.likeStatus = likeStatus;
    }

    @Override
    public String toString() {
        return String.format(
                "RemoveLikeResponse[userId='%s', shopId='%s', likeStatus='%s']",
                userId, shopId, likeStatus);
    }
}

File: AngularBoot/src/app/pages/_models/shop/removeLike/removeLikeResponse.ts
import {RemoveLikeStatus} from './removeLikeStatus';

export class RemoveLikeResponse {

    private userId: string;
    private shopId: string;
    private likeStatus: RemoveLikeStatus;

    public getUserId(): String {
        return this.userId;
    }

    public setUserId(userId: string) {
        this.userId = userId;
    }

    public getShopId(): string {
        return this.shopId;
    }

    public setShopId(shopId: string) {
        this.shopId = shopId;
    }

    public getLikeStatus(): RemoveLikeStatus {
        return this.likeStatus;
    }

    public setLikeStatus(likeStatus: RemoveLikeStatus) {
        this.likeStatus = likeStatus;
    }
}
